src/data/full-quiz.js

```diff
@@ -1,4 +1,76 @@
+    Distractor1: "null",
+    Distractor2: "undefined",
+    Distractor3: "document",
+    Explanation: "In global execution context, this keyword always points to window object",
+    Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this",
+  },
+  {
+    Question: "Which queue is emptied first by Javascript event loop?",
+    Answer: "Microtask queue",
+    Distractor1: "Callback queue (Macrotask queue)",
+    Distractor2: "None",
+    Distractor3: "Both",
+    Explanation: "Event loop first empties the Microtask queue and once its empty, starts to empty Callback queue",
+    Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/EventLoop",
+  },
+  {
+    Question: "Which of the following is not in Temporal Dead Zone(TDZ) during hoisting?",
+    Answer: "var",
+    Distractor1: "let",
+    Distractor2: "const",
+    Distractor3: "class",
+    Explanation: "var is hoisted with default value of undefined. While let, const and classes are hoisted but are in Temporal Dead Zone(TDZ) until declaration is executed.",
+    Link: "https://developer.mozilla.org/en-US/docs/Glossary/Hoisting",
+  },
+  {
+    Question: "Which of the following expression will return true in Javascript?",
+    Answer: "null >= 0",
+    Distractor1: "null == 0",
+    Distractor2: "null === 0",
+    Distractor3: "null > 0",
+    Explanation: "The >= operator will convert null to 0, which will return true. But in == operator, null only equals to undefined. Because of this null <= will also be true.",
+    Link: "https://stackoverflow.com/questions/2910495/why-null-0-null-0-but-not-null-0",
+  },
+  {
+    Question: "Which of the following is not part of Javascript?",
+    Answer: "console",
+    Distractor1: "Array",
+    Distractor2: "Object",
+    Distractor3: "let, var, const",
+    Explanation: "console is part of a web API which is provided by browsers and nodejs.",
+    Link: "https://developer.mozilla.org/en-US/docs/Web/API/console",
+  },
+  {
+    Question: "Which of the following method returns a function?",
+    Answer: "bind",
+    Distractor1: "apply",
+    Distractor2: "call",
+    Distractor3: "None of given",
+    Explanation: "The bind functions returns a new function with given arguement as the new function's this keyword.",
+    Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Function/bind",
+  },
+  {
+    Question: "What is a first class function?",
+    Answer: "The ability to send and receive functions like value.",
+    Distractor1: "Functions with higher priority than others.",
+    Distractor2: "Functions declared in classes",
+    Distractor3: "None of given",
+    Explanation: "If functions can be sent and received just like values, than that languge is said to have first class functions. They are also called first class citizens.",
+    Link: "https://www.geeksforgeeks.org/difference-between-first-class-and-higher-order-functions-in-javascript/",

```

Comment:
For the link field we only want to use freeCodeCamp articles or official docs.

Proposed fix:
Link:"https://www.freecodecamp.org/news/discover-the-power-of-first-class-functions-fd0d7b599b69/"